apps/trading/components/market-volume/market-volume.tsx

```diff
@@ -57,8 +57,8 @@ export const MarketVolume = ({ marketId }: { marketId: string }) => {
   });
 
   return (
-    <HeaderStat heading={t('Volume')}>
-      <div data-testid="trading-volume">{marketVolume}</div>
+    <HeaderStat heading={t('Volume')} testId="market-volume">
```

Comment:
what is the difference between `data-testid` and `testId`?

Reply:
I need to pass somehow data-testid into the HeaderStat component, but I cannot name it `data-testid` because it's variable here. So I just create variable, pass the value into component and then use it as html attribute